x/evidence/abci.go

```diff
@@ -0,0 +1,25 @@
+package evidence
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k Keeper) {
```

Comment:
nit missing godoc. Moving the comment below here suffices

Reply:
updated.